src/main/scala/org/globalforestwatch/summarystats/treecoverloss/TreeLossTile.scala

```diff
@@ -21,8 +21,8 @@ case class TreeLossTile(loss: TreeCoverLoss#ITile,
                         grossCumulAbovegroundRemovalsCo2: GrossCumulAbovegroundRemovalsCo2#OptionalFTile,
                         grossCumulBelowgroundRemovalsCo2: GrossCumulBelowgroundRemovalsCo2#OptionalFTile,
                         netFluxCo2: NetFluxCo2e#OptionalFTile,
-                        grossEmissionsCo2eNonCo2: GrossEmissionsNonCo2Co2e#OptionalFTile,
-                        grossEmissionsCo2eCo2Only: GrossEmissionsCo2OnlyCo2e#OptionalFTile,
+                        grossEmissionsCo2eNonCo2: GrossEmissionsNonCo2Co2eBiomassSoil#OptionalFTile,
```

Comment:
Why is this biomass soil now?

Reply:
Those emissions are called biomasssoil now because I added three new layers that are emissions for soil_only (as opposed to biomass + soil emissions). So I added biomasssoil to distinguish them from the soil_only emissions. In the carbonflux package, I need to distinguish between biomasssoil and soil_only but treecoverloss only uses biomasssoil. Is it okay to leave this change for clarity, though?